examples/azure-stack/kubernetes-azurestack.json

```diff
@@ -44,9 +44,9 @@
                 ]
             }
         },
-        "servicePrincipalProfile": {
-            "clientId": "",
-            "secret": ""
+        "servicePrincipalProfile": {	
+            "clientId": "",	
+            "secret": ""	
```

Comment:
Please clean up the spacing here. Probably could just `git checkout` the file. Doesn't look like you made any changes.